doc/sphinx/addendum/sprop.rst

```diff
@@ -0,0 +1,225 @@
+.. _sprop:
+
+SProp (proof irrelevant propositions)
+=====================================
+
+.. warning::
+
+   The status of strict propositions is experimental.
+
+This section describes the extension of |Coq| with definitionally
+proof irrelevant propositions (types in the sort :math:`\SProp`, also
+known as strict propositions). To use :math:`\SProp` you must pass
+``-allow-sprop`` to the |Coq| program or use :opt:`Allow SProp`.
+
+.. opt:: Allow SProp
+   :name: Allow SProp
+
+   Allows using :math:`\SProp` when set and forbids it when unset. The
+   initial value depends on whether you used the command line
+   ``-allow-sprop``.
+
+.. coqtop:: none
+
+   Set Allow SProp.
+
+Some of the definitions described in this document are available
+through ``Coq.Logic.StrictProp``, which see.
```

Comment:
I'm not sure I understand what "which see" means. Is that an unfinished sentence?

Reply:
It means "cf the thing I just mentioned"

Reply:
If we could make the module name a link to the stdlib doc that would work, but I'm not sure how feasible that is.

Reply:
(it seems to be used significantly by emacs lisp and C people, see language tab in https://github.com/search?q=%22which+see%22&type=Code)